plugins/CMakeLists.txt

```
@@ -204,17 +215,22 @@ endif()
 # 4. build DFHack as normal. The plugins you added will be built as well.
 
 if(NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/external/CMakeLists.txt")
-    file(WRITE "${CMAKE_CURRENT_SOURCE_DIR}/external/CMakeLists.txt"
+    set(content_str
 "# Add external plugins here - this file is ignored by git
 
 # Recommended: use add_subdirectory() for folders that you have created within
 # this folder, or dfhack_plugin() for single files that you have added here.
 
 # See the end of /plugins/CMakeLists.txt for more details.
 ")
+    SUBDIRLIST(SUBDIRS external)
```

```suggestion
    subdirlist(SUBDIRS external)
```
(probably requires adjusting the macro definition too)